cou/apps/auxiliary.py

```diff
@@ -128,6 +128,15 @@ def channel_codename(self) -> OpenStackRelease:
 
         track: str = self._get_track_from_channel(self.channel)
         compatible_os_releases = TRACK_TO_OPENSTACK_MAPPING[(self.charm, self.series, track)]
+
+        if not compatible_os_releases:
+            raise ApplicationError(
+                f"Channel: {self.channel} for charm '{self.charm}' on series '{self.series}' is "
+                f"currently not supported in this tool. Please take a look at the documentation: "
```

Comment:
nit, to avoid committing to supporting weird channels in the future :D
```suggestion
                f"not supported by COU. Please take a look at the documentation: "
```